app/helpers/Formatters.scala

```diff
@@ -39,9 +39,12 @@ object Formatters {
     s"${DateTimeFormatter.ISO_DATE_TIME.format(dateTime.truncatedTo(ChronoUnit.SECONDS))}Z"
   }
 
+  private val kbMin: Long = 1000
```

Comment:
Move line 42 and 43 after line 31 as these vals must be together (if these are not inside ant def)